src/translations/en.global.json

```diff
@@ -0,0 +1,12 @@
+{
+    "account.requestCode.success": "Sent SMS with code.",
+    "account.requestCode.error": "Failed to send SMS with code.",
+    "account.setupAccountRecovery.success": "Success",
+    "account.setupAccountRecovery.error": "Failed to setup account recovery.",
+    "account.recoverAccount.success": "Success",
+    "account.recoverAccount.error": "Failed to recover account.",
+    "account.create.success": "Congrats! this name is available.",
```

Comment:
"Congrats! This name is available."

Reply:
Feel free to edit copy directly next time